core/src/main/java/org/infinispan/manager/impl/ReplicableRunnableCommand.java

```diff
@@ -48,12 +48,6 @@ public boolean isReturnValueExpected() {
       return false;
    }
 
-   @Override
-   public boolean canBlock() {
```

Comment:
Actually I forgot about this... This is part of the ClusterExecutor which was not converted to be non blocking, so these indeed required some changes. You should be able to change the `invokeAsync` method to offload the call to the blocking manager though.

https://issues.redhat.com/browse/ISPN-11482 which was never completed.

Reply:
It looks like the other function was already converted at https://github.com/infinispan/infinispan/commit/7dceec11949f7bbb1ce699341161e32841c3839a#diff-e0572be74ffdf8f1762b718032655f06b8e838aa77530b84f1596ef04511fc7f. So it just needs the same treatment.

Reply:
I just noticed that `ReplicableRunnableCommand` is also not propagating the subject, like `ReplicableManagerFunctionCommand` is doing. I'll add that too